controllers/istio/istio_controller.go

```diff
@@ -98,6 +99,13 @@ func validate(istio *v1alpha1.Istio) error {
 	if istio.Spec.Version == "" {
 		return reconciler.NewValidationError("spec.version not set")
 	}
+	istioVersion, err := semver.NewVersion(istio.Spec.Version)
+	if err != nil {
+		return reconciler.NewValidationError("spec.version is not a valid semver: " + err.Error())
+	}
+	if config.Config.MaximumIstioVersion != nil && istioVersion.GreaterThan(config.Config.MaximumIstioVersion) {
+		return reconciler.NewValidationError("spec.version is not supported")
```

Comment:
Can you add the version to the error message. This makes it read as though setting the field itself is not supported.

Reply:
done!